src/components/CohortBuilder/ParticipantsTableView/SaveSetModal.tsx

```diff
@@ -196,7 +195,11 @@ const SaveSetModal: FunctionComponent<Props> = (props) => {
   useEffect(() => {
     const defaultName = filtersToName({ filters: sqon });
     setLoadingDefaultTagName(true);
-    setDefaultTagName(defaultName !== SET_DEFAULT_NAME ? defaultName : '');
+    const formDefaultValue =
+      saveSetActionType === SaveSetActionsTypes.CREATE
+        ? { nameSet: defaultName !== SET_DEFAULT_NAME ? defaultName : '' }
+        : { nameSet: setToRename?.name || '' };
```

Comment:
maybe use `??` instead of `||`

`??` is a logical operator that returns its right-hand side operand when its left-hand side operand is null or undefined, and otherwise returns its left-hand side operand. Whereas `||` returns its right-hand side operand when its left-hand side is false (so not only null or undefined)